ddp-workspace/projects/ddp-rgp/src/styles/_activity.scss

```diff
@@ -377,3 +377,44 @@ ddp-activity-dropdown-picklist-question select {
     padding: 0;
   }
 }
+
+/**
+ * Enrollment nested activities
+ */
+.instance {
+  .instance__divider {
+    display: block;
+    border-top: 2px solid #0C457D;
+    border-bottom: none !important;
+    border-color: #0C457D !important;
+    margin: 25px 0 10px 0;
+  }
+
+  .instance__header {
+    font-size: 1.1rem;
+    font-weight: 500;
+    line-height: 1.6rem;
+    margin-top: 20px;
+  }
+}
+
+.instance:first-child .instance__divider {
+  display: none;
+}
+
+.instance:first-child .instance__header {
+  display: none;
+}
```

Comment:
This is necessary to hide the header label for the first instance (this is something new for embedded nested activities!).

Before:
![rgp_bug_label_before](https://user-images.githubusercontent.com/5713833/116942178-1d55a000-ac3f-11eb-8cdb-44cf7ec9c41a.png)

After:
![rgp_bug_label_after](https://user-images.githubusercontent.com/5713833/116942184-221a5400-ac3f-11eb-9c45-9602d08e2434.png)